utils/validate.ts

```diff
@@ -0,0 +1,28 @@
+interface ErrorInterface {
+  valid: boolean;
+  error: null | string;
+}
+
+export const invalidCharError: ErrorInterface = {
+  valid: false,
+  error: 'contains invalid characters'
+};
+export const validationSuccess = { valid: true, error: null };
+export const usernameTooShort = { valid: false, error: 'is too short' };
+export const usernameIsHttpStatusCode = {
+  valid: false,
+  error: 'is a reserved error code'
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+export const isNumeric = (num: any) => !isNaN(num);
```

Comment:
It doesn't look like this is used outside of this file, so it's fine to use `isNaN` inline.
```suggestion
```